flask_app/backend_jobs/ingestion_pipeline/main.py

```diff
@@ -55,13 +55,17 @@ def _is_valid_image(image):
     """ This function returns whether the given image satisfies minimum requirements of the platform
     e.g. url != none
     """
-    return \
-        image.url and \
-        image.latitude and \
-        image.longitude and \
-        image.format and \
-        image.width_pixels > 100 and \
-        image.height_pixels > 100
+    return image.url
```

Comment:
make the logic explicit and hence more readable: `return image.url is not None`

Reply:
changed the return